site/docs/concepts/web-app.md

```diff
@@ -198,7 +212,8 @@ You use the table to monitor your materializations. It's nearly identical to the
 
 When you click **View Details** for a materialization, the **Status** and **Specification** viewers are revealed.
 
-The **Status** window shows the full identifier of the shard that backs your materialization. If there's an error, you can view its logs.
+The **Status** window shows the full identifier of the shard(s) that backs your materialization.
```

Comment:
See the comment left behind in the Captures page section regarding the _Status_ section of the details panel.